src/applications/appeals/995/containers/ConfirmationPage.jsx

```diff
@@ -71,36 +72,43 @@ export class ConfirmationPage extends React.Component {
             className="usa-button screen-only"
             onClick={window.print}
           >
-            Print this page
+            Print this confirmation
           </button>
         </div>
         <h3>What to expect next</h3>
         <p>
-          If we need more information, we'll contact you to tell you what other
-          information you'll need to submit. We'll also tell you if we need to
+          If we need more information, we’ll contact you to tell you what other
+          information you’ll need to submit. We’ll also tell you if we need to
           schedule an exam for you.
         </p>
         <p>
-          When we've completed your review, we'll mail you a decision packet
+          When we’ve completed your review, we’ll mail you a decision packet
           with the details of our decision.{' '}
           <a href="/decision-reviews/after-you-request-review/">
             Learn more about what happens after you request a review
           </a>
           .
         </p>
-        <h3>What to do if you have questions now</h3>
         <p>
-          You can call us at <va-telephone contact="8008271000" /> (
-          <va-telephone contact="711" tty />) We're here Monday through Friday,
-          8:00 a.m. to 9:00 p.m. ET.
+          If you requested a decision review and haven’t heard back from us yet,
+          please don’t request another review. Call us instead.
         </p>
         <p>
-          Note: If you requested a decision review and haven't heard back from
-          us yet, please don't request another review. Call us instead.
+          Note: You can choose to have a hearing at any point in the claims
+          process. Contact us online through Ask VA to request a hearing.{' '}
+          <a href="https://ask.va.gov/">Contact us through Ask VA</a>
+        </p>
+        <p>
+          You can also call us at <va-telephone contact="8008271000" /> (
+          <va-telephone contact="711" tty />
+          ).
         </p>
         <a href="/track-claims/your-claims" className="usa-button">
           Track the status of your claim
         </a>
+        <p />
```

Comment:
what's the `<p />` here for?

Reply:
Add space above the `H3`... I guess I could have added a margin to the `H3` as well